src/core/syscalls/hint_code.rs

```diff
@@ -46,3 +46,11 @@ pub(crate) const GET_BLOCK_TIMESTAMP: &str =
 
 pub(crate) const GET_BLOCK_NUMBER: &str =
     "syscall_handler.get_block_number(segments=segments, syscall_ptr=ids.syscall_ptr)";
+
+// *************************
```

Comment:
ADDR_BOUND_PRIME and ADDR_IS_250 are not syscall hints, the previous one like GET_BLOCK_NUMBER is

Reply:
Right, I'd missed it. Fixed.